src/styles/components/_related-components.scss

```diff
@@ -45,14 +43,11 @@ $breakpoint-move-arrow-left: 38.125rem; // 610px/16
         filter: invert(1);
       }
 
-      @media (min-width: $breakpoint-move-arrow-left) {
-        right: 85%;
-        width: 2.875rem; // 46px/16
-        height: 3.5625rem; // 57px/16
-      }
-
       @media (min-width: $breakpoint-large-screen-860) {
-        right: 50%;
+        width: 2.8rem; // 45px/16
+        height: 3.4rem; // 54px/16
+        left: 10%;
```

Comment:
Did we want to use consistent units across the breakpoints? During a previous PR, Cat mentioned that percentages can be awfully small depending on the screen size, so she recommended going with rems (but of course, as I type this, maybe she was referring to just widths/heights....)

Reply:
That seems like a good idea! I removed the left property from this query entirely so that it'll inherit a 2rem value.